CalibFormats/CaloObjects/src/CaloSamples.cc

```diff
@@ -3,14 +3,13 @@
 #include <math.h>
 #include <iostream>
 
-const int CaloSamples::MAXSAMPLES;
-
```

Comment:
I don't really feel a lot of responsibility for ECAL code... it would be nice if someone would go through ECAL, trigger, etc. code and fix all instances of inconsistent hardcoding like this, but it's not high on my todo list.